draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/decision-tokens.scss

```diff
@@ -0,0 +1,22 @@
+@import "~@kaizen/design-tokens/sass/color";
+
+// Option tokens for Heart theme background colours.
+$ot-color-background-slate-10: rgba(var(--kz-var-color-slate-rgb-params), 0.1);
```

Comment:
I like this naming, I'm finding it pretty easy to reason about.

Reply:
I am a bit confused about seeing both "option token" and "background" to refer to the same thing. My understanding was that option tokens are devoid of meaning (like background colour) and decision tokens are what assigns an option token to a meaning. Have I got a weird take?

Reply:
I hadn't seen these comments and went and refactored to try and make it clearer.

@ActuallyACat can you take another look?

Reply:
@sebpearce your understanding is correct, what I'm doing is just a bit weird, and the naming was off - this was just supposed to represent Slate (as defined by a CSS custom property) with 10% alpha. A true option token shouldn't have had the word "background". Perhaps `ot-color-slate-opacity-10`?

Either way I've had a go at refactoring so the raw CSS variable names are not used outside of the decision tokens file (rather, a mixin exists that you add to the component root which defines the css variables that the decision tokens depend on)